src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.DivisionByZeroException;
+import core.basesyntax.exception.IllegalOperatorException;
+import core.basesyntax.exception.InvalidValuesException;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final int DEFAULT_A = 5;
+    private static final int DEFAULT_B = 17;
+    private static final double ADD_A_B = 22;
+    private static final double SUBTRACT_A_B = -12;
+    private static final double MULTIPLY_A_B = 5 * 17;
+    private static final double DIVIDE_A_B = (double) 5 / 17;
+    private static final double DIVIDE_B_A = (double) 17 / 5;
+    private static final double RAISE_A_TO_B = Math.pow(5, 17);
+    private static final double RAISE_B_TO_A = Math.pow(17, 5);
+    private static final int ZERO = 0;
+    private Calculator calculator = new Calculator();
+
+    @Test
+    void add_Ok() {
+        double expected = ADD_A_B;
+        double actual = calculator.calculate(DEFAULT_A, DEFAULT_B, '+');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void add_NotOk() {
+        assertThrows(InvalidValuesException.class, () -> {
+            calculator.calculate(Integer.MAX_VALUE, 1, '+');
+        });
+    }
+
+    @Test
+    void subtract_Ok() {
```

Comment:
```suggestion
    void subtraction_Ok() {
```
Let's add a full word to others too